pkg/kubectl/cmd/testdata/edit/testcase-edit-from-empty/0.response

```diff
@@ -0,0 +1,9 @@
+{
+    "apiVersion": "v1",
+    "items": [],
+    "kind": "List",
+    "metadata": {
+        "resourceVersion": "",
+        "selfLink": ""
+    }
+}
```

Comment:
just noticed this stub doesn't actually match what the API server returns for this API call. It should be an actual capture of an API response:
```
{
    "kind": "ConfigMapList",
    "apiVersion": "v1",
    "metadata": {
        "selfLink": "/api/v1/namespaces/default/configmaps",
        "resourceVersion": "252"
    },
    "items": []
}
```